src/Windows/Host/Client/Impl/BrokerServices/WebServer.cs

```diff
@@ -135,8 +134,7 @@ public class WebServer {
             var remoteUri = new Uri(remoteUrl);
             var localUri = new UriBuilder(remoteUri);
 
-            WebServer server;
-            if(!Servers.TryGetValue(remoteUri.Port, out server)) {
+            if (!Servers.TryGetValue(remoteUri.Port, out WebServer server)) {
```

Comment:
Is it possible that because of concurrent access we end up with two `WebServer` instances on the same port?